bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hawkular-ui-services",
   "description": "WIP:Angular Services for the Hawkular Console.",
-  "version": "0.8.16",
+  "version": "0.8.17",
```

Comment:
We don't version this manually but with 'bower version patch' that increments the version, and creates the tags and keeps them in sync.